Source/Initialization/PlasmaInjector.cpp

```diff
@@ -189,6 +189,14 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
         pp.getarr("num_particles_per_cell_each_dim", num_particles_per_cell_each_dim);
 #if WARPX_DIM_XZ
         num_particles_per_cell_each_dim[2] = 1;
+#endif
+#if WARPX_DIM_RZ
+        if (num_particles_per_cell_each_dim[1]<2*WarpX::n_rz_azimuthal_modes){
```

Comment:
According to you PR description, shouldn't this be
```suggestion
        if (num_particles_per_cell_each_dim[1]<=2*WarpX::n_rz_azimuthal_modes){
```
?

Reply:
Thanks for the review @MaxThevenet !
I think that because when the if condition is verified, the code is aborted with the message below.
And we want the code to run for #parts in theta =2*WarpX::n_rz_azimuthal_modes.
That we should keep it as it is.